README.md

```diff
@@ -122,6 +122,19 @@ module.exports = {
 };
 ```
 
+## Modify Prettier Config
+
+You can customize Prettier configuration to your requirements. In example, you can update `prettier.config.js` with new `semi` value.
```

Comment:
I think this conflicts with `eslint-config-zillow` 🤔

Reply:
Ahh, it does. Auto-fix on save works, but then I get error from ESLint. I can try another setting or I can mention to use in tandem with ESLint config.

Reply:
Yeah, I think in general eslint config is used and then the prettier file is just a redirect:
https://github.com/zillow/javascript/tree/latest/packages/eslint-plugin-zillow#prettier-editor-plugin-integration

Reply:
Got it. I'll remove this section and info about modifying rules.